### PR TITLE
completed challenge for mergeLists

### PR DESCRIPTION
**Pull request by tcoots88**
- [x] Top-level README “Table of Contents” is updated
 - [ x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [x] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: code401challenges/README.md [hunk 1]
```diff
@@ -64,3 +64,21 @@ Determine the mid point by first declaring the the start and end of an array.  D
...
+
+## Challenge Description
+<!-- Description of the challenge -->
+A function called mergeLists which takes two linked lists as arguments. The two linked lists zip together into one so that the nodes alternate between the two lists and return a reference to the head of the zipped list. 
+## Approach & Efficiency
+<!-- What approach did you take? Why? What is the Big O space/time for this approach? -->
+Traverse each linked list. Point the next node of current node in list1 to next node in list2
+## Solution
+<!-- Embedded whiteboard image -->
+![Whiteboard for mergeLists](whiteboard.jpg)
```
Comment (JamesDansie) [inline code401challenges/README.md]: Update the whiteboard link to;
src/main/resources/whiteboard.jpg
